server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/JmapRfc8621Configuration.scala

```diff
@@ -33,12 +35,11 @@ object JmapRfc8621Configuration {
   val URL_PREFIX_PROPERTIES: String = "url.prefix"
 
   val UPLOAD_LIMIT_PROPERTIES: String = "upload.max.mb.size"
-  val MB: Long =  1024 * 1024
-  val UPLOAD_LIMIT_30_MB: Long = 30
-  var MAXSIZE_UPLOAD: Option[Size] = None
+  val UPLOAD_LIMIT_30_MB: String = "30M"
```

Comment:
Do not parse a default value

Reply:
Any suggestion from you?

Reply:
Use a long: `val UPLOAD_LIMIT_30_MB: Long = 30 * 1024 * 1024` for the default value (already parsed)

Reply:
Well
Size object have only one way to create by `parse(String)`.If we dont have configuration which cause nullpointerException when parse.
I would like `parse` either configuration provided or default one than handle error and switching value at other place

Reply:
And another thing is: 
with your request `upload.max.size=30MB` will cause error: `java.lang.NumberFormatException: For input string: "30M"`
because of enum unit are only have:
```
private enum Unit {
        NoUnit,
        B,
        K,
        M,
        G
    }
```
So you cant you `MB`

Reply:
Ok for M vs MB

However 

`Use a long: val UPLOAD_LIMIT_30_MB: Long = 30 * 1024 * 1024 for the default value (already parsed)` still stands ;-)